src/AzureAuth.Test/Ado/TokenFormatterTest.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace AzureAuth.Test.Ado
+{
+    using FluentAssertions;
+
+    using Microsoft.Authentication.AzureAuth.Ado;
+
+    using NUnit.Framework;
+
+    internal class TokenFormatterTest
+    {
+        [Test]
+        public void Header_Bearer()
+        {
+            TokenFormatter.HeaderBearer("foobar").Should().Be("Authorization: Bearer foobar");

```

Comment:
Somehow, I wonder if a string extension method `.WithHeaderBearer` is more fluent.

Reply:
This is an option that might fit nicely with a recommendation Tucker had while pairing.

Reply:
Now if only I could remember what that recommendation was... 😆